dht-node/src/index.ts

```diff
@@ -3,6 +3,7 @@ import { startDHT } from '@/dht_node/index'
 
 // config
 import CONFIG from './config'
+import DEVOP from './config/devop'
```

Comment:
Same suggestion as on top, in backend we don't allow this anymore but we do some named export so we can write:
```suggestion
import { CONFIG } from './config'
import { DEVOP } from './config/devop'
```

Reply:
as mentioned above this notation doesn#t work for the config-path. I don't know why?